rllib/algorithms/apex_ddpg/tests/test_apex_ddpg.py

```diff
@@ -23,7 +23,7 @@ def test_apex_ddpg_compilation_and_per_worker_epsilon_values(self):
         config = (
             apex_ddpg.ApexDDPGConfig()
             .rollouts(num_rollout_workers=2)
-            .reporting(min_sample_timesteps_per_reporting=100)
+            .reporting(min_sample_timesteps_per_iteration=100)
```

Comment:
but should we move these out of reporting() though, if they mean more than reporting???